contracts/tokens/Cruzo1155.sol

```diff
@@ -4,13 +4,15 @@ pragma solidity ^0.8.6;
 import "./ERC1155URI.sol";
 
 contract Cruzo1155 is ERC1155URI {
-    using Counters for Counters.Counter;
-    Counters.Counter private _tokenIds;
     address public marketAddress;
 
     string public name;
     string public symbol;
 
+    mapping (uint256 => address) private tokenIdToCreator;
+
+    uint256 private lastCreated;
```

Comment:
`lastCreated` is also not used

Reply:
last created is used via total, and it is reached via tests